<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addinterview</class>
 <widget class="QDialog" name="addinterview">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="Interview">Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>491</width>
     <height>331</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="ai_cancle">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Abbrechen</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ai_type">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>211</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Interview Typ</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="ai_ok">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>230</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sprache</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QComboBox" name="ai_language">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ai_name">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>441</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>431</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Interview Typ
Job: Beantworte 7 Fragen zu deinem Beruf und Werdegang.
Student: Beantworte 8 Fragen zu deinem Studium und Werdegang.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>ai_name</tabstop>
  <tabstop>ai_type</tabstop>
  <tabstop>ai_language</tabstop>
  <tabstop>ai_ok</tabstop>
  <tabstop>ai_cancle</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
